forms-web-app/src/app.js

```diff
@@ -5,19 +5,38 @@ const path = require('path');
 const cookieParser = require('cookie-parser');
 const nunjucks = require('nunjucks');
 const dateFilter = require('nunjucks-date-filter');
+const session = require('express-session');
 
-const indexRouter = require('./routes/index');
+const applicationNameRouter = require('./routes/application-name');
+const applicationNumberRouter = require('./routes/application-number');
+const checkAnswersRouter = require('./routes/check-answers');
 const eligibilityRouter = require('./routes/eligibility');
+const indexRouter = require('./routes/index');
 const taskListRouter = require('./routes/task-list');
+const yourDetailsRouter = require('./routes/your-details');
 
 const app = express();
 
+const sessionConfig = {
+  secret: process.env.SESSION_KEY,
+  resave: false,
+  saveUninitialized: true,
+  cookie: {},
+};
+
+if (app.get('env') === 'production') {
+  app.set('trust proxy', 1); // trust first proxy
+  sessionConfig.cookie.secure = true; // serve secure cookies
```

Comment:
might not be correct, copied wholesale from the npm docs

Reply:
I'd prefer not relying on `production` - would prefer having a `config.server.useSecureSessionCookie` boolean that I can control instead of `if (app.get('env') === 'production') { ... }`

For the most part, any HTTP headers for production will be set in the nginx-ingress section so you don't need to worry about that.

Reply:
OK, does this mean I can get rid of :

`app.set('trust proxy', 1); // trust first proxy` ? 

I have made the other changes as requested. 

One thing to flag: https://github.com/foundry4/appeal-planning-decision/pull/75/commits/9281a716ab82f670be6f56a28341ce9c02e55d68#diff-9d4e2ee2e03d9f5247e0df2662fb1491e6e09dcbc72c6e84c481f0f9dcf1e395R10

Not sure if that's the best way. All env vars are strings afaik, so an explicit check for `"true"` is the best I could come up with.

Reply:
Yeah, that's exactly the way that works

FYI, if it's a boolean then I either use `{ item: process.env.BOOL === 'true' }` (if you want it to default to false or `{ item: process.env.BOOL !== 'false' }` (if you want it to default to false. I think defaulting this to `false` is the correct way to go about it